analyzing/duration_analyzer.py

```diff
@@ -0,0 +1,95 @@
+"""
+This module provides functionality to analyze alert durations from JSON data.
+
+Functions:
+    __get_data(path: str) -> dict:
+        Reads JSON data from a file and returns it as a dictionary.
+        Args:
+            path (str): The directory path where the JSON file is located.
+        Returns:
+            dict: The JSON data loaded from the file.
+        Raises:
+            FileNotFoundError: If the JSON file does not exist.
+
+    __calc_mean_duration_per_alertname(results: dict) -> dict:
+        Calculates the mean duration for each alert name from the given results.
+        Args:
+            results (dict): The dictionary containing alert data.
+        Returns:
+            dict: A dictionary with alert names as keys and their mean durations as values.
+
+    __write_data(data: dict, out: str):
+        Writes the given data to a JSON file.
+        Args:
+            data (dict): The data to be written to the file.
+            out (str): The output file path.
+
+    get_mean_duration_per_alertname(path: str, out: str) -> dict:
+        Main function to get the mean duration per alert name and write the results to a file.
+        Args:
+            path (str): The directory path where the input JSON file is located.
+            out (str): The output file path where the results will be written.
+        Returns:
+            dict: A dictionary with alert names as keys and their mean durations as values.
+"""
+
+import json
+import logging
+import os
+
+logger = logging.getLogger("alertmagnet")
+
+
+def __get_data(path: str):
```

Comment:
__get_data could be many things, how about a little bit more specific like:

`__read_json_file` 

Should this go to utilities then?